proposals/lip-0041.md

```diff
@@ -39,8 +39,12 @@ We define the following constants:
 
 | Name               | Type    | Value       | Description                             |
 | ------------------ |---------| ------------| ----------------------------------------|
-|`MODULE_ID_AUTH`    | uint32  | TBD         | ID of the Auth module.                  |
+|`MODULE_ID_AUTH`    | bytes  | TBD         | ID of the Auth module.                  |
 |`STORE_PREFIX_AUTH` | bytes   | 0x0000      | Store prefix of the auth data substore. |
+|`COMMAND_ID_REGISTER_MULTISIGNATURE_GROUP` | bytes   | 0x0000      | Command ID of the register multisignature group command. |
+|`TYPE_ID_MULTISIGNATURE_GROUP_REGISTERED` | bytes   | 0x0000      | Type ID of the signature group registered event. |
```

Comment:
If I understand well the Events LIP, for each module the type_ID value 0x0000 is allocated for the standard event which indicates the success of the transaction (constant `EVENT_TYPE_ID_STANDARD`). If I am not missing something, this means that the value of `TYPE_ID_MULTISIGNATURE_GROUP_REGISTERED` should change..

Reply:
Good catch, changed.